packages/api/src/module/write/email-confirmation/domain/requestEmailConfirmation.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { requestEmailConfirmationCommand } from '@/module/commands/request-email-conformation';
+
+import { requestEmailConfirmation } from './requestEmailConfirmation';
+
+describe('Request email confirmation', () => {
+  it('create event EmailConfirmationWasRequested when requestEmailConfirmation is invoked', () => {
+    const exampleUserId = 'ca63d023-4cbd-40ca-9f53-f19dbb19b0ab';
```

Comment:
I think we don't need `example` before every value. 
Sometimes i follow approach like: 
```
const given = {
userId: ....
confirmationToken: ...
```

if you'd like, but just userId is enough. We know that we are in scope of test.